src/Constants_Conversion.f90

```diff
@@ -30,24 +30,20 @@ MODULE Constants_Conversion
   IMPLICIT NONE
   PRIVATE
 
-  REAL(SRK),PUBLIC,PARAMETER ::   ZERO=0.000000000000000_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::    ONE=1.000000000000000_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::    TWO=2.000000000000000_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::  QTRPI=7.853981633974483E-1_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER :: HALFPI=1.570796326794897E+0_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::     PI=3.141592653589793E+0_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER ::  TWOPI=6.283185307179586E+0_SRK !16 digits
-  REAL(SRK),PUBLIC,PARAMETER :: FOURPI=1.256637061435917E+1_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::    ZERO=0.000000000000000_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::     ONE=1.000000000000000_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::     TWO=2.000000000000000_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::   QTRPI=7.853981633974483E-1_SRK !16 digits
+  REAL(SRK),PUBLIC,PARAMETER ::  HALFPI=1.570796326794897E+0_SRK !16 digits
```

Comment:
Any hurt in making other multiplies of PI actual multiples:  ```HALFPI=TWO*PI```, ```PI=TWO*HALFPI```, ```TWOPI=TWO*PI```, ```FOURPI=TWO*TWOPI```.  Since these constants are direct binary operation (I think its a shift... but I could be wrong) it shouldn't incur much roundoff.